addons/l10n_es_edi_tbai_pos/static/src/app/add_tbai_refund_reason_popup/add_tbai_refund_reason_popup.js

```diff
@@ -0,0 +1,19 @@
+import { Dialog } from "@web/core/dialog/dialog";
+import { usePos } from "@point_of_sale/app/store/pos_hook";
+import { Component, useState } from "@odoo/owl";
+
+export class AddTbaiRefundReasonPopup extends Component {
+    static template = "l10n_es_edi_tbai_pos.AddTbaiRefundReasonPopup";
+    static components = { Dialog };
+
+    setup() {
+        this.pos = usePos();
+        this.state = useState({
+            l10n_es_tbai_refund_reason: this.props.order.l10n_es_tbai_refund_reason || "R1",
```

Comment:
This is the default one in the widget¸ so it is the default refund_reason

We can change which one it should be

But if I don't put `|| R1` here and the user confirm the reason directly, no reasons will be set (from what I've tested).  Like because the user didn't actually select anything, no reason is set (even if it is displayed as selected)